lib/iwdp.js

```diff
@@ -0,0 +1,78 @@
+import { SubProcess } from 'teen_process';
+import EventEmitter from 'events';
+import { execSync } from 'child_process';
+import Promise from 'bluebird';
+import logger from './logger';
+
+const request = Promise.promisify(require('request'));
+
+const IWDP_CMD = 'ios_webkit_debug_proxy';
+const MAX_RETRIES = 10;
+
+class IWDP extends EventEmitter {
+
+  constructor (udid = null) {
```

Comment:
Isn't the 'webkitDebugProxyPort' configuration instructing Appium which port `ios_webkit_debug_proxy` is opened at?

I think in this case, because the driver is opening ios_webkit_debug_proxy we want to set it to a fixed port.